controllers/storagecluster/external_resources_test.go

```diff
@@ -114,8 +114,9 @@ func createExternalClusterReconcilerFromCustomResources(
 		ObjectMeta: metav1.ObjectMeta{
 			Name: "ocsinit",
 			Annotations: map[string]string{
-				UninstallModeAnnotation: string(UninstallModeGraceful),
-				CleanupPolicyAnnotation: string(CleanupPolicyDelete),
+				UninstallModeAnnotation:                     string(UninstallModeGraceful),
+				CleanupPolicyAnnotation:                     string(CleanupPolicyDelete),
+				util.BackwardCompatabilityInfoAnnotationKey: string(util.JsonMustMarshal(util.BackwardCompatabilityInfo{Pre4_19DeploymentMode: util.ExternalDeploymentMode})),
```

Comment:
This is too long to be a single line.
```suggestion
				util.BackwardCompatabilityInfoAnnotationKey: string(
					util.JsonMustMarshal(
						util.BackwardCompatabilityInfo{
							Pre4_19DeploymentMode: util.ExternalDeploymentMode
						}
					)
				),
```